### PR TITLE
feat: add schema for event_screen_view

### PR DESCRIPTION
**Pull request by irchanbani**

### CODE REVIEW

File: schemas/event_screen_view_bq_schema.json [hunk 1]
```diff
@@ -0,0 +1,254 @@
...
+        "description": "Timestamp when the event occurred."
+    },
+    {
+        "name": "user_id",
+        "type": "STRING",
+        "mode": "REQUIRED",
+        "description": "ID of the user."
+    },
+    {
+        "name": "user_join",
```
Comment (aongko) [inline schemas/event_screen_view_bq_schema.json]: user_join_time or user_join_date (please consult with Edo)

Comment (irchanbani) [reply schemas/event_screen_view_bq_schema.json]: Edo said that he prefer `user_join` than the other two

File: variables.tf [hunk 2]
```diff
@@ -0,0 +1,8 @@
+variable "project" {
+  description = "The project to deploy to, if not set the default provider is used"
```
Comment (aongko) [inline variables.tf]: what do you mean by "provider" in `the default "provider" is used`

File: main.tf [hunk 3]
```diff
@@ -0,0 +1,12 @@
+resource "google_bigquery_table" "event_screen_view" {
```
Comment (aongko) [inline main.tf]:
```suggestion
resource "google_bigquery_table" "screen_view" {
```

**Pull request merged**